.github/workflows/deploy-main-docs.yml

```diff
@@ -18,7 +18,6 @@ jobs:
     - name: Set-up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: "2.5.1"
         bundler-cache: true
```

Comment:
If no version is specified, always use the latest? or how can we specify the ruby version 3.2.1?

Reply:
we're using [docs/.ruby-version](https://github.com/bitmakerla/estela/pull/149/files#diff-6ba93206e23de0ffd8dec0668fd330b937a08cd13a4db0cd8e64c969ed3a8e69) to get the ruby version.